### PR TITLE
HELM-297: add a stack for testing Helm

### PR DESCRIPTION
**Pull request by RangerRick**
This PR adds a stack for Helm testing which also comes with some scripts to pre-populate data.

https://issues.opennms.org/browse/HELM-297

### CODE REVIEW

**Review by indigo423 — APPROVED** [review]
LGTM! Nice that we have now tools to test flows :) The only thing, I would probably would rename it to something like `helm-flow-stack` cause it adds elastic and the whole flow support to this specific stack.

File: helm-stack/helm-test/create-fake-flows.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	java -version 2>&1 | grep version
+	exit 1
+fi
+
+echo "downloading populator jar"
+wget --quiet -c -O /tmp/nephron-testing-bundled.jar https://www.opennms.com/~ranger/helm-test/nephron-testing-bundled-0.3.2-SNAPSHOT.jar
+
+echo "creating fake flows in elasticsearch"
+java -jar /tmp/nephron-testing-bundled.jar --esRawFlowOutput=ELASTIC_SEARCH
+curl -XPUT -H 'Content-Type: application/json' http://localhost:9200/_template/netflow -d@netflow-template.json
```
Comment (swachter) [inline helm-stack/helm-test/create-fake-flows.sh:15]: I would setup the index template first. I am not sure how ES behaves when index definitions come after the samples.

```suggestion
echo "setup index template for netflow index"
curl -XPUT -H 'Content-Type: application/json' http://localhost:9200/_template/netflow -d@netflow-template.json
echo "creating fake flows in elasticsearch"
java -jar /tmp/nephron-testing-bundled.jar --esRawFlowOutput=ELASTIC_SEARCH
```

**Review by swachter — CHANGES_REQUESTED** [review]
Looks great, but I think we are not yet there.

When testing, OpenNMS did exit because of a file permission problem:

```
04:32:01.006 [Main] WARN  org.opennms.install.Installer - Could not create file: /opt/opennms/etc/libraries.properties
04:32:01.137 [Main] ERROR org.opennms.install.Installer - OpenNMS is configured to run as 'opennms' but '/opt/opennms/deploy/.gitkeep' is not writable by that account.
04:32:01.137 [Main] ERROR org.opennms.install.Installer - To fix permissions, run '/opt/opennms/bin/fix-permissions' as root
```

General thoughts:

* Can the `README.md` file moved up one folder?
* Can the `grafana-test.json` file renamed into `opennms-requisition.json`?

**Comment by RangerRick** [comment]
leaving the branch for now until I get confirmation from grafana folks taht they're done with it

**Pull request merged**